luarules/gadgets/unit_dgun_behaviour.lua

```diff
@@ -6,7 +6,7 @@ function gadget:GetInfo()
 		desc = "D-Gun projectiles hug ground, deterministic damage against Commanders",
 		author = "Anarchid, Sprung",
 		layer = 0,
-		enabled = false -- Disabled for now because d-gun has been replaced with EMP weapon
+		enabled = true -- Disabled for now because d-gun has been replaced with EMP weapon
```

Comment:
```suggestion
		enabled = enabled
```
Otherwise it will be enabled for all games, not just testing the com updates. Also change line 1 to
`local enabled = Spring.GetModOptions().comupdate`, but Github doesn't let you comment on unchanged lines.